gravitee-apim-console-webui/src/entities/application/Application.ts

```diff
@@ -41,6 +40,10 @@ export interface Application {
   origin?: string;
 }
 
+export interface AdditionalClientMetadata {
```

Comment:
just a note on the commit naming, it is a bit weird to me to have this typescript file (and the following one) in a commit named `feat(rest-api)`

Reply:
This will be squashed into single commit.